templates/etc/systemd/system/caddy.service.erb

```diff
@@ -5,24 +5,52 @@ After=network-online.target
 Wants=network-online.target systemd-networkd-wait-online.service
 
 [Service]
+Restart=on-abnormal
```

Comment:
while you heavily patch the module? Should we also add a statement like `this file is managed by puppet`?

Reply:
Done